src/utils/url.ts

```diff
@@ -23,5 +23,6 @@ const getTransactionIdFromRequestParams = (req: Request): string => req.params[c
 const getSubmissionIdFromRequestParams = (req: Request): string => req.params[config.ROUTE_PARAM_SUBMISSION_ID];
 
 export const isRemoveJourney = (req: Request): boolean => {
-  return req.query[config.JOURNEY_QUERY_PARAM] === config.JourneyType.remove;
+  return req.query !== undefined &&
```

Comment:
Not sure why this has changed. Might not be required after you make the changes above.

Reply:
Removed change from PR, put this in because at one point the typescript was uneasy and needed reassuring that the undefined scenario was being handled, but doe snot seem to be an issue now.